library/src/main/java/org/apache/apex/malhar/lib/state/spillable/Spillable.java

```diff
@@ -44,8 +45,10 @@
   }
 
   /**
-   * This represents a spillable {@link java.util.Map}. The underlying implementation
-   * of this map uses the serialized representation of the key object as a key. User's that receive an
+   * This represents a spillable {@link java.util.Map}. The implementations assume
+   * that the fields from objects of type K that are serialized are the same fields that are used in the
```

Comment:
I think the comment be simplified using K1 and K2. The assumption is that k1.equals(k2) should be consistent with  k1.toByteArray().equals(k2.toByteArray())